ecommerce/api_test.py

```diff
@@ -817,10 +817,7 @@ def test_validate_basket_different_product(basket_and_coupons):
     )
     with pytest.raises(ValidationError) as ex:
         validate_basket_for_checkout(basket_and_coupons.basket)
-    assert (
-        ex.value.args[0]["runs"]
-        == "Some runs present in basket which are not part of product"
-    )
```

Comment:
The validation should remain in place while the queryset should be updated to include any other eligible runs/products.